frontend/public/storage/components/storage-overview/storage-overview.jsx

```diff
@@ -180,6 +201,7 @@ export class StorageOverview extends React.Component {
           },
           ...utilizationData,
           alertsResponse,
+          ...topConsumersData,
```

Comment:
please do not send such generic props to the consumer. i.e. `stats` and `loaded`

Reply:
ack